config/locales/en.yml

```diff
@@ -48,8 +48,12 @@ en:
     student_loan_country: "When you applied for your student loan where was your home address?"
     student_loan_how_many_courses: "How many higher education courses have you studied?"
     student_loan_start_date:
-      one_course: "When did the first year of your higher education course start?"
-      two_or_more_courses: "When did the first year of each higher education course start?"
+      one_course:
+        question: "When did the first year of your higher education course start?"
+        validation_error: "Select when the first year of your higher education course started"
```

Comment:
Seems a bit weird to nest this inside `questions`. Do we need another hash at the same level?